tracer/src/Datadog.Trace.Tools.Runner/Checks/Resources.cs

```diff
@@ -33,6 +33,11 @@ internal static class Resources
         public const string AspNetCoreProcessNotFound = "Could not find the ASP.NET Core applicative process.";
         public const string VersionConflict = "Tracer version 1.x can't be loaded simultaneously with other versions and will produce orphaned traces. Make sure to synchronize the Datadog.Trace NuGet version with the installed automatic instrumentation package version.";
 
+        public const string TracingWithBundleProfilerPath = "Assuming process is traced with Datadog.Trace.Bundle Nuget, configuration(CORECLR_PROFILER_PATH or COR_PROFILER_PATH) was found for this service: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/dotnet-core/?tab=nuget#install-the-tracer";
+        public const string TracingWithInstaller = "Assuming process is traced with Installer/MSI, Nuget configuration not found: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/dotnet-core/?tab=containers#install-the-tracer";
```

Comment:
I'm not sure if these two messages are super valuable. Can we remove them or do you see a strong reason to keep them?

Reply:
Some users may accidentally not follow the right docs, so based on the reasoning for the change I commented [here](https://github.com/DataDog/dd-trace-dotnet/pull/4605/files/ee81c2ed803ed30ee079a98da079902c0c6ec1ad#r1320282395), I also added this comments to guide customers and support about what the customer is trying to do so that they are able to hit the link and confirm that the setup they are trying to do matches the documentation, do you think it might not be as useful?

Reply:
Well if everything's OK, then this output might be unnecessary to the user since instrumentation is already hooked up correctly. But if it's not OK, then I think this could be useful to redirect to the docs, but maybe we could break it up into the following:
1. Whether we detected the per-application Datadog.Trace.Bundle NuGet package in the application output.
2. If the user is trying to instrument with the per-application Datadog.Trace.Bundle NuGet package, print a link to the .NET Core or .NET Framework page with the NuGet tab.
3. Otherwise, if the user is trying to instrument with the platform package, print a link to the .NET Core or .NET Framework page with the Windows/Linux tab.

What do you think of that sort of guidance?